Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -27,6 +27,7 @@ namespace RDKit {
 
 RDKIT_FILEPARSERS_EXPORT std::string strip(const std::string &orig);
 
+namespace v2 {
```

Comment:
I'm not sure of this change of moving the exception into the v2 namespace.   This might break the api,  a user might have used:

```
try {
} catch FileParsers::MolFileUnhandledFeatureException {
}
```

Which would no longer compile.

Reply:
Now, I expect this has never been used this way...

Reply:
This exception is handled in the mol file parser itself (in `MolFromMolDataStream()`) and should never escape to client code